themes/src/main/resources/theme/keycloak.v2/login/resources/css/styles.css

```diff
@@ -26,6 +37,10 @@ div.kc-logo-text {
     margin: 0 auto;
 }
 
+.kc-card {
+    border-top: 4px solid var(--keycloak-card-top-color);
+}
+
```

Comment:
I must say I personally like this little detail, but don't we want to sick more to the way it was designed by PF and refrain from adding customization? https://v5-archive.patternfly.org/components/login-page/react/basic/

Reply:
If we are going to overwrite existing PatternFly conventions, let's not alias variables and use the original class (if possible). Any custom CSS that we add will make the eventual upgrade to a new PatternFly version more difficult, so we try to avoid it as much as possible.

If the intention here is to have a visual separator, there is a [Divider](https://v5-archive.patternfly.org/components/divider) component that we could use.

Reply:
Using the divider instead is not possible?

```html
<hr class="pf-v5-c-divider" />
```

Reply:
Ahhh, no, thanks for clarifying. I was under the impression that this was part of the separator between the form elements, consider this resolved.